R/CellGraphAssay.R

```diff
@@ -115,6 +115,14 @@ CreateCellGraphAssay <- function (
   ...
 ) {
 
+  # Unlist dots
+  dots <- list(...)
+  if ("command" %in% names(dots)) {
+    command <- dots[["command"]]
+  } else {
+    command = "CreateCellGraphAssay"
+  }
+
```

Comment:
Yes you're right! I forgot to implement that solution.

Regarding your question about the `options("pixelatorR.auto_cleanup")` (I cannot reply to the general comment :-/), when it is checked within `edgelists_directories_clean` it automatically resorts to TRUE if it is NULL. Maybe this is a bad idea since it will run the cleanup without asking. What I'm worried about is if someone runs an analysis workflow and then it stops because of the prompt. But maybe it's better to be conservative here?

"ReadMPX_Seurat does not trigger edgelists_directories_clean, perhaps it should?"
Yeah maybe. My reasoning was that this command is usually only run at the beginning of an analysis workflow. But I guess it doesn't hurt to add it there as well.

Reply:
I think it would be optimal if the cleanup could run safely without prompt. But at this stage, it is not completely benign? I agree with you that it would be annoying if the workflow stops because of the prompt. Not sure what is best here.

I checked some of my edgelist files today, and I think it would be good if `ReadMPX_Seurat` also can trigger cleanup. In my edgelist folder of a project I had 16GB of edgelists that were generated only by `ReadMPX_Seurat` because of closing and opening the R project many times.

Reply:
I think we can keep the prompt as default. If you manage develop a whole workflow you can probably easily figure out how to modify the global variables appropriately.

Ok I'll add the check to `ReadMPX_Seurat`.